freeze_unfreeze/contracts/FreezeUnfreeze.sol

```diff
@@ -0,0 +1,68 @@
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/security/Pausable.sol";
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
+
+contract FreezeUnfreeze is ERC20, Ownable, Pausable{
+    using SafeMath for uint256;
+
+    // freeze accounts mapping
+    mapping(address=>bool) frozen;
+
+    constructor() ERC20("MyToken", "MTK") {}
+
+    function freeze(address[] calldata addrs) external onlyOwner{
+        for (uint256 i = 0; i < addrs.length; ++ i) {
+            frozen[addrs[i]] = true;
```

Comment:
Check 0x00000... is not in the list, otherwise owner will not be able to mint/burn